toolkit/templates/forms/_checkbox-tree-selection-buttons.html

```diff
@@ -1,30 +1,30 @@
-<fieldset class="checkbox-tree__selection-buttons">
+<fieldset class="checkbox-tree__inputs" id="checkbox-tree__inputs">
```

Comment:
Setting the `id` like this does mean you're not going to be able to use the macro more than once on a page (and have it still be compliant)